stdlib/LinearAlgebra/test/pinv.jl

```diff
@@ -63,69 +63,54 @@ function tridiag(T::Type, m::Integer, n::Integer)
 end
 tridiag(m::Integer, n::Integer) = tridiag(Float64, m::Integer, n::Integer)
 
-function randn_float64(m::Integer, n::Integer)
-    a=randn(m,n)
-    b = Matrix{Float64}(undef, m, n)
-    for i=1:n
-        for j=1:m
-            b[j,i]=convert(Float64,a[j,i])
-        end
-    end
-    return b
-end
-
-function randn_float32(m::Integer, n::Integer)
-    a=randn(m,n)
-    b = Matrix{Float32}(undef, m, n)
-    for i=1:n
-        for j=1:m
-            b[j,i]=convert(Float32,a[j,i])
-        end
-    end
-    return b
-end
-
+function test_pinv(a,tol1,tol2)
+    m,n = size(a)
 
-function test_pinv(a,m,n,tol1,tol2,tol3)
     apinv = @inferred pinv(a)
-
+    @test size(apinv) == (n,m)
     @test norm(a*apinv*a-a)/norm(a) ≈ 0 atol=tol1
-    x0 = randn(n); b = a*x0; x = apinv*b
-    @test norm(a*x-b)/norm(b) ≈ 0 atol=tol1
-    apinv = pinv(a,sqrt(eps(real(one(eltype(a))))))
+    @test norm(apinv*a*apinv-apinv)/norm(apinv) ≈ 0 atol=tol1
+    for _ in 1:100
+        b = a*randn(n)
+        x = apinv*b
+        @test norm(a*x-b)/norm(b) ≈ 0 atol=tol1
+    end
```

Comment:
Why the loop here? Is this intended or something left over from editing? Same below in lines 83-87.

Reply:
Since `randn(n)` is included, I thought it would be better to verify multiple times rather than just one test.

Reply:
I think just one test should suffice. If we do this everywhere, the tests will take way too long.

Reply:
Okay, I have removed `for _ in 1:100`.